Documentation/observability/metrics.rst

```diff
@@ -48,7 +48,7 @@ use ``operator.prometheus.enabled=true``.
      --set operator.prometheus.enabled=true
 
 The ports can be configured via ``prometheus.port``,
-``proxy.prometheus.port``, or ``operator.prometheus.port`` respectively.
+``envoy.prometheus.port``, or ``operator.prometheus.port`` respectively.
```

Comment:
Why change this? It seems reasonable enough to keep as-is...

Reply:
@squeed the change from `proxy` to `envoy` here at this specific place in the docs or in general?

Overall it's kind of the chance to name Envoy explicitly and cleanup the naming mess with other (L7-)proxies - and therefore moving the small existing `proxy.prometheus.*` part too

Reply:
gotcha, makes sense.